src/server/identity/server/api_server.go

```diff
@@ -79,7 +138,14 @@ func (a *apiServer) GetIDPConnector(ctx context.Context, req *identity.GetIDPCon
 	}, nil
 }
 
-func (a *apiServer) UpdateIDPConnector(ctx context.Context, req *identity.UpdateIDPConnectorRequest) (resp *identity.UpdateIDPConnectorResponse, retErr error) {
+func (a *apiServer) UpdateIDPConnector(ctx context.Context, req *identity.UpdateIDPConnectorRequest) (*identity.UpdateIDPConnectorResponse, error) {
+	if a.env.Config.IdentityConnectors != "" {
+		return nil, errors.New("identity.UpdateIDPConnector() is disable when the Identity Connectors are set via environment.")
```

Comment:
"is disabled" (this typo appears in a few other places; was going to mention each but realized there are at least 4 and it already sounds kind of mean once :))

Reply:
yikes good catch